src/redux/settings.ts

```diff
@@ -102,15 +155,15 @@ export const settingsChangeNativeCurrency = nativeCurrency => async dispatch =>
 };
 
 // -- Reducer --------------------------------------------------------------- //
-export const INITIAL_STATE = {
+export const INITIAL_STATE: SettingsState = {
   accountAddress: '',
   chainId: 1,
   language: 'en',
   nativeCurrency: 'USD',
   network: networkTypes.mainnet,
```

Comment:
we can use the enum: `Network.mainnet` and then remove the import of `networkTypes`